examples/icons/index.html

```diff
@@ -4,7 +4,7 @@
     <base data-trunk-public-url/>
     <meta charset="utf-8"/>
     <title>Yew App</title>
-    <link rel="stylesheet" href="bootstrap-icons-v1.10.5/bootstrap-icons.css"/>
+    <link rel="stylesheet" href="bootstrap-icons-v1.11.3/bootstrap-icons.css"/>
```

Comment:
Perhaps we should not put the version in the directory; it is already in the header of the file.

Reply:
I disagree with myself now re: caching